docs/kubernetes.adoc

```diff
@@ -74,7 +74,12 @@ You can configure Kubernetes by using any of the next configuration properties i
 |===
 | Options                             | Type           | Env | Description                                                   
 | kubernetes.master                   | URL            | Any | The URL to the Kubernetes master                               
-| kubernetes.domain                   | String         | OSE | Domain to use for creating routes for services                 
+| name                                | String         | Any | Username to log in server
```

Comment:
Why not username?

Reply:
We are looking for system property and env variable. If we used username, then `System.getenv("USERNAME")` returns your host name.

I tried user.name as well, but `System.getProperty("user.name")` returns your host name. thats why I am using name

Reply:
ok, then leave it as is.